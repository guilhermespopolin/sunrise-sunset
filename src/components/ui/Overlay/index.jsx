import React from 'react'

import Box from 'components/ui/Box'

const Overlay = props => (
  <Box
    display="flex"
    alignItems="center"
    justifyContent="center"
    position="absolute"
    top="0"
    right="0"
    bottom="0"
    left="0"
    backgroundColor="rgba(0, 0, 0, 0.8)"
    {...props}
  />
)

export default Overlay
