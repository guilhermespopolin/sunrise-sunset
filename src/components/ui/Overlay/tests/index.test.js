import React from 'react'
import { render } from '__tests__/utils/rtlWrapper'

import Overlay from '..'

describe('<Overlay />', () => {
  it('should render properly', () => {
    const { container } = render(<Overlay />)

    expect(container.firstChild).toMatchSnapshot()
  })
})

