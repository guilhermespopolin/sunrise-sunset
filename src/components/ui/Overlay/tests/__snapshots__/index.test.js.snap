// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Overlay /> should render properly 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  background-color: rgba(0,0,0,0.8);
  position: absolute;
  top: 0;
  right: 0;
  bottom: 0;
  left: 0;
}

<div
  class="c0"
  display="flex"
/>
`;
